engine/src/main/java/org/terasology/rendering/dag/nodes/FinalPostProcessingNode.java

```diff
@@ -104,9 +105,9 @@ public void initialise() {
     public void process() {
         PerformanceMonitor.startActivity("rendering/finalPostProcessing");
 
-        ocUndistorted = frameBuffersManager.getFBO("ocUndistorted");
-        sceneFinal = frameBuffersManager.getFBO("sceneFinal");
-        sceneOpaque = frameBuffersManager.getFBO("sceneOpaque");
+        ocUndistorted = dynamicFBM.getFBO(OC_UNDISTORTED_URN);
+        sceneFinal = dynamicFBM.getFBO(DefaultDynamicFBOs.Final.getResourceUrn());
+        sceneOpaque = dynamicFBM.getFBO(DefaultDynamicFBOs.ReadOnlyGBuffer.getResourceUrn());
```

Comment:
I didn't want to the enum's have actual FBO reference. Just names. I can do that if you think that's better.